#! /bin/sh /usr/share/dpatch/dpatch-run
## 18-xorg-1.6.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use XServer-Xorg 1.6

@DPATCH@

--- virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/x11/vboxmouse/Makefile.kmk.org	2009-09-09 20:07:47.000000000 +0200
+++ virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/x11/vboxmouse/Makefile.kmk	2009-09-10 19:48:05.000000000 +0200
@@ -27,7 +27,6 @@
 # vboxmouse_drv
 #
 if1of ($(KBUILD_TARGET), linux)
- SYSMODS += vboxmouse_drv
  vboxmouse_drv_TEMPLATE = VBOXGUESTR3XF86MOD
  vboxmouse_drv_DEFS.linux = linux
  vboxmouse_drv_DEFS.x86 += __i386__
@@ -61,7 +60,6 @@
 #
 # vboxmouse_drv_70
 #
-DLLS += vboxmouse_drv_70
 vboxmouse_drv_70_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxmouse_drv_70_DEFS = \
 	XFree86Server IN_MODULE XFree86Module XFree86LOADER XINPUT XORG_7X
@@ -82,7 +80,6 @@
 #
 # vboxmouse_drv_71
 #
-DLLS += vboxmouse_drv_71
 vboxmouse_drv_71_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxmouse_drv_71_DEFS := $(vboxmouse_drv_70_DEFS)
 vboxmouse_drv_71_INCS := \
@@ -99,7 +96,6 @@
 #
 # vboxmouse_drv_14
 #
-DLLS += vboxmouse_drv_14
 vboxmouse_drv_14_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxmouse_drv_14_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
 vboxmouse_drv_14_INCS := \
@@ -116,7 +112,6 @@
 #
 # vboxmouse_drv_15
 #
-DLLS += vboxmouse_drv_15
 vboxmouse_drv_15_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxmouse_drv_15_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
 vboxmouse_drv_15_INCS := \
@@ -131,23 +126,20 @@
 #
 # vboxmouse_drv_16
 #
-DLLS += vboxmouse_drv_16
-vboxmouse_drv_16_TEMPLATE = VBOXGUESTR3XORGMOD
-vboxmouse_drv_16_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
-vboxmouse_drv_16_INCS := \
-	$(VBOX_PATH_X11_XORG_1_6) \
-	$(VBOX_PATH_X11_XORG_1_6)/X11 \
-	$(VBOX_PATH_X11_ROOT)/xorg-server-1.6.0 \
-	$(VBOX_PATH_X11_ROOT)/xorg-server-1.6.0-local \
-	$(PATH_SUB_CURRENT)
-vboxmouse_drv_16_SOURCES = \
+DLLS += vboxmouse_drv
+vboxmouse_drv_TEMPLATE = VBOXGUESTR3XORGMOD
+vboxmouse_drv_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
+vboxmouse_drv_INCS := \
+	/usr/include/x11 \
+	/usr/include/xorg \
+	/usr/include/pixman-1
+vboxmouse_drv_SOURCES = \
 	vboxmouse_15.c
 
 
 #
 # vboxmouse_drv_17
 #
-DLLS += vboxmouse_drv_17
 vboxmouse_drv_17_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxmouse_drv_17_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
 ## @todo replace $(VBOX_PATH_X11_ROOT)/xorg-server-1.6.0-local
--- virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/x11/vboxvideo/Makefile.kmk.org	2009-09-09 20:07:50.000000000 +0200
+++ virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/x11/vboxvideo/Makefile.kmk	2009-09-10 19:51:47.000000000 +0200
@@ -26,7 +26,6 @@
 # vboxvideo_drv
 #
 if1of ($(KBUILD_TARGET), linux)
- SYSMODS += vboxvideo_drv
  vboxvideo_drv_TEMPLATE = VBOXGUESTR3XF86MOD
  vboxvideo_drv_DEFS.linux = linux
  vboxvideo_drv_DEFS.x86 = __i386__
@@ -113,7 +112,6 @@
 #         base keywords instead of using .solaris or .linux.
 #         Also it is *important* to use := and not = when deriving a properity.
 #
-DLLS += vboxvideo_drv_70
 vboxvideo_drv_70_TEMPLATE = VBOXGUESTR3XORGMOD
 if1of ($(KBUILD_TARGET), linux)
  vboxvideo_drv_70_CFLAGS += \
@@ -139,7 +137,6 @@
 #
 # vboxvideo_drv_71
 #
-DLLS += vboxvideo_drv_71
 vboxvideo_drv_71_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxvideo_drv_71_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
 vboxvideo_drv_71_DEFS := $(vboxvideo_drv_70_DEFS)
@@ -158,7 +155,6 @@
 #
 # vboxvideo_drv_13
 #
-DLLS += vboxvideo_drv_13
 vboxvideo_drv_13_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxvideo_drv_13_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
 vboxvideo_drv_13_DEFS := $(vboxvideo_drv_70_DEFS)
@@ -180,7 +176,6 @@
 # This uses the same code (vboxvideo_13.c) as 1.3, but is built
 # for 1.4 as well in case there should be any relevant header changes.
 #
-DLLS += vboxvideo_drv_14
 vboxvideo_drv_14_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxvideo_drv_14_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
 vboxvideo_drv_14_DEFS := $(vboxvideo_drv_70_DEFS)
@@ -199,7 +194,6 @@
 #
 # vboxvideo_drv_15
 #
-DLLS += vboxvideo_drv_15
 vboxvideo_drv_15_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxvideo_drv_15_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
 vboxvideo_drv_15_DEFS := $(vboxvideo_drv_70_DEFS) NO_ANSIC PCIACCESS \
@@ -220,19 +214,17 @@
 #
 # vboxvideo_drv_16
 #
-DLLS += vboxvideo_drv_16
-vboxvideo_drv_16_TEMPLATE = VBOXGUESTR3XORGMOD
-vboxvideo_drv_16_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
-vboxvideo_drv_16_DEFS := $(vboxvideo_drv_15_DEFS)
-vboxvideo_drv_16_INCS = \
-	$(VBOX_PATH_X11_XORG_1_6) \
-	$(VBOX_PATH_X11_XORG_1_6)/X11 \
-	$(VBOX_PATH_X11_ROOT)/glproto-1.4.9 \
-	$(VBOX_PATH_X11_ROOT)/libdrm-2.4.5 \
-	$(VBOX_PATH_X11_ROOT)/xf86driproto-2.0.4 \
-	$(VBOX_PATH_X11_ROOT)/xorg-server-1.6.0 \
-	$(VBOX_PATH_X11_ROOT)/xorg-server-1.6.0-local
-vboxvideo_drv_16_SOURCES  = \
+DLLS += vboxvideo_drv
+vboxvideo_drv_TEMPLATE = VBOXGUESTR3XORGMOD
+vboxvideo_drv_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
+vboxvideo_drv_DEFS := $(vboxvideo_drv_15_DEFS)
+vboxvideo_drv_INCS = \
+	/usr/include/xorg \
+	/usr/include/x11 \
+	/usr/include/pixman-1 \
+	/usr/include/X11/dri \
+	/usr/include/drm
+vboxvideo_drv_SOURCES  = \
 	vboxvideo_15.c \
 	vboxutils.c \
 	vboxvideo_dri.c
@@ -241,7 +233,6 @@
 #
 # vboxvideo_drv_17
 #
-DLLS += vboxvideo_drv_17
 vboxvideo_drv_17_TEMPLATE = VBOXGUESTR3XORGMOD
 vboxvideo_drv_17_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
 vboxvideo_drv_17_DEFS := $(vboxvideo_drv_15_DEFS)
--- virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/common/crOpenGL/Makefile.kmk.org	2009-09-09 20:07:35.000000000 +0200
+++ virtualbox-ose-3.0.6-dfsg/src/VBox/Additions/common/crOpenGL/Makefile.kmk	2009-09-10 23:04:18.000000000 +0200
@@ -69,21 +69,16 @@
 VBoxOGL_INCS           = .
 if1of ($(KBUILD_TARGET),linux solaris)
  VBoxOGL_INCS     += \
-	$(VBOX_PATH_X11_ROOT)/libXdamage-1.1 \
-	$(VBOX_PATH_X11_ROOT)/libXcomposite-0.4.0 \
-	$(VBOX_PATH_X11_ROOT)/libXfixes-4.0.3 \
-	$(VBOX_PATH_X11_ROOT)/damageproto-1.1.0 \
-	$(VBOX_PATH_X11_ROOT)/compositeproto-0.4 \
-	$(VBOX_PATH_X11_ROOT)/fixesproto-4.0 \
-	$(VBOX_PATH_X11_ROOT)/libx11-1.1.5-other \
-	$(VBOX_PATH_X11_ROOT)/1.3/xorg \
+	/usr/include/x11 \
+	/usr/include/xorg \
+	/usr/include/pixman-1 \
 	$(VBOX_PATH_MESA_SOURCE) \
 	$(VBOX_PATH_MESA_SOURCE)/include \
 	$(VBOX_PATH_MESA_SOURCE)/src/mesa/ \
 	$(VBOX_PATH_MESA_SOURCE)/src/mesa/glapi \
 	$(VBOX_PATH_MESA_SOURCE)/src/mesa/main \
 	$(VBOX_PATH_MESA_SOURCE)/src/mesa/drivers/dri/common \
-	$(PATH_ROOT)/src/VBox/Additions/x11/x11include/libdrm-2.4.5
+	/usr/include/drm
  VBoxOGL_DEFS     += VBOX_NO_NATIVEGL
 endif
 ifdef VBoxOGL_DRI
